compile_proto.sh

```diff
@@ -10,6 +10,11 @@ printf "DONE\n\n"
 
 file="./protos/fishjam/peer_notifications.proto"
 
-printf "Compiling: file $file\n"
-protoc --plugin=./node_modules/.bin/protoc-gen-ts_proto --ts_proto_out=./src/ $file
-printf "DONE\n"
+printf "Compiling: file $file for react-native\n"
```

Comment:
we don't need this for RN

Reply:
Why it is no longer required?

Reply:
it was required when rn client handled websockets communication
this code generates this stuff: https://github.com/fishjam-cloud/react-native-client-sdk/blob/3606a391982ed0314f084a30edfe0977b14a307b/src/protos/fishjam/peer_notifications.ts which was used for example here: https://github.com/fishjam-cloud/react-native-client-sdk/blob/3606a391982ed0314f084a30edfe0977b14a307b/src/FishjamClientContext.tsx#L103
I got rid of it recently and now the native sdks handle it

Reply:
removed